src/api/transaction/suspay-finish.js

```diff
@@ -0,0 +1,48 @@
+/* @flow */
+'use strict';
+const _ = require('lodash');
+const utils = require('./utils');
+const validate = utils.common.validate;
+const Transaction = utils.common.core.Transaction;
+
+function createSusPayFinishTransaction(account, payment) {
+  validate.address(account);
+  validate.suspendedPaymentFinish(payment);
+
+  const transaction = new Transaction();
+  transaction.susPayFinish({
+    from: account,
```

Comment:
I think adding `from` might've been a mistake. We'll probably never get rid of it, but I'd use `account` here instead